.circleci/config.yml

```diff
@@ -928,6 +932,8 @@ jobs:
   osx_package:
     macos:
       xcode: "10.1.0" # 10.1.0 is macOS 10.13.6
+    environment:
+      ERLANG_ROCKSDB_BUILDOPTS: "-j2"
```

Comment:
this runs in VM and I don't think you need that, that would slow the builds probably